en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_delete_multi_world=Delete Sub-world
-gui.xaero_delete_multi_world_msg1=Would you like to delete all waypoints data for the selected sub-world?
-gui.xaero_delete_multi_world_msg2=This action cannot be reverted. Make sure you know what you're doing.
-gui.xaero_always_show_distance=Always Disp. WP Dist.
+gui.xaero_delete_world_msg1=Would you like to delete all waypoint data from the selected world/server?
+gui.xaero_delete_world_msg2=This action cannot be undone! Make sure you know what you are doing.
+gui.xaero_delete_multi_world=Delete Sub-World
+gui.xaero_delete_multi_world_msg1=Would you like to delete all waypoint data from the selected sub-world?
+gui.xaero_delete_multi_world_msg2=This action cannot be undone! Make sure you know what you are doing.
+gui.xaero_always_show_distance=Always Disp. Dist. to WP
 gui.xaero_item_tooltip=Item Tooltip
 gui.xaero_entity_tooltip_settings=Item Tooltip Settings
 gui.xaero_item_tooltip_min_lines=Minimum Lines
-gui.xaero_item_tooltip_time=Show For
-gui.xaero_player_names=Player Head Names
+gui.xaero_item_tooltip_time=Display For
+gui.xaero_player_names=Player Names Under Heads
 gui.xaero_multiply_all_by_8=Sub-World Coordinates * 8
 gui.xaero_divide_all_by_8=Sub-World Coordinates / 8
-gui.xaero_multiply_msg1=This option should only be used to fix incorrect waypoint coordinates.
-gui.xaero_multiply_msg2=Are you sure you would like to MULTIPLY all sub-world coordinates by 8?
-gui.xaero_divide_msg2=Are you sure you would like to DIVIDE all sub-world coordinates by 8?
+gui.xaero_multiply_msg1=This option should be used only to fix incorrect waypoint coordinates.
+gui.xaero_multiply_msg2=Are you sure you would like to multiply all sub-world coordinates by 8?
+gui.xaero_divide_msg2=Are you sure you would like to divide all sub-world coordinates by 8?
 gui.xaero_share=Share
-gui.xaero_share_msg1=Are you sure you would like to share this waypoint with §cEVERYONE§f in chat?
-gui.xaero_share_msg2=Please make sure you don't expose a secret location.
-gui.xaero_roof=Roof
+gui.xaero_share_msg1=Are you sure you would like to share this waypoint with §cEVERYONE§f in the chat?
+gui.xaero_share_msg2=Make sure you do not expose a secret location!
+gui.xaero_roof=Ceiling
 gui.xaero_partial=Partial
 gui.xaero_full=Full
-gui.xaero_show_light_level=Show Light Level
+gui.xaero_show_light_level=Display Light Level
 gui.xaero_up=[Go Up]
 gui.xaero_down=[Go Down]
 gui.xaero_display_all_sets=Toggle All WP Sets Render
+gui.xaero_waypoint_name=waypoint name
 gui.xaero_render_layer=Overlay Render Layer
 gui.xaero_show_time=Display Game Time
 gui.xaero_24h=24h
 gui.xaero_12h=12h
 gui.xaero_temporary=temporary
 gui.xaero_restore=Restore
-gui.xaero_biomes_vanilla=Biomes In Vanilla Mode
-gui.xaero_potion_effects_blink=Blink When Low
+gui.xaero_biomes_vanilla=Biomes in Vanilla Color Mode
+gui.xaero_potion_effects_blink=Blink When Runs Out
```

Comment:
I think so. I took it from the Minecraft Wiki from this article: https://minecraft.gamepedia.com/Java_Edition_1.9. You can use Ctrl + F (does it work in modern non-Firefox browsers? IDK) to search for "Effects that are about to run out blink" for more context.

Reply:
Yeah but "when about to run out" isn't exactly "when runs out". I would have no problem with the first one besides being too long.

Reply:
Well, AFAIK an alternative form of "when about to run out" is "when is running out". I had to shorten it a bit to make it fit, so since it is also a repetitive action, I changed Present Continuous to Present Simple, and achieved a label that fits, but I'm not entirely sure whether I could do so or not. "Run out" is a phrasal verb, so it may have different rules.

Reply:
Yeah, idk, it really seems like it means running out completely to me. I think I'll trust your decision here though.

Reply:
Thank you for your trust then :)